src/lib.rs

```diff
@@ -1603,15 +1606,13 @@ impl RtcConfig {
     }
 
     /// The configured DtlsCert.
-    ///
-    /// The certificate is uniquely created per new RtcConfig.
-    pub fn dtls_cert(&self) -> &DtlsCert {
-        &self.dtls_cert
+    pub fn dtls_cert(&self) -> Option<&DtlsCert> {
+        self.dtls_cert.as_ref()
     }
 
     /// Set DTLS certification.
     pub fn set_dtls_cert(mut self, dtls_cert: DtlsCert) -> Self {
-        self.dtls_cert = dtls_cert;
+        self.dtls_cert = Some(dtls_cert);
```

Comment:
Some doc here be good too. That unless you set something, it will be automatically created for you.